src/app/shared/account-header/account-header.component.scss

```
@@ -0,0 +1,39 @@
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+// Uses the same primary color token as mat-toolbar color="primary"
+// Light: #1074b9  |  Dark: #5ba2ec  (via --md-sys-color-primary)
+.header {
+  background-color: var(--md-sys-color-primary, #1074b9) !important;
+  color: var(--md-sys-color-on-primary, #fff) !important;
+
+  // Override Material Design subtitle / title colors within the header
+  /* stylelint-disable-next-line selector-max-universal */
+  ::ng-deep .mat-mdc-card-title,
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)